### PR TITLE
Ab/cb106  Refactoring the `<ContainerSections/>` component

### PR DESCRIPTION
**Pull request by Andresbl123**
The <ContainerSections /> component currently handles logic that should not be inside it, as its primary function is to be a layout component. Business logic should not reside in such a component.

To improve the separation of responsibilities, it is necessary to refactor the <ContainerSections /> component. So the button sections could be separated into another component, which receives what is needed to work through props, complementing the work done on the cb-105 board.

### CODE REVIEW

**Review by Angelbt12 — APPROVED** [review]

File: src/components/layout/ContainerSections/Stocktray/index.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  };
+}
+
+interface IStocktrayProps {
+  isMobile?: boolean;
+  actionButtons?: IActionButtons;
+  navigation: () => void;
+}
+
+export const Stocktray = (props: IStocktrayProps) => {
```
Comment (carin91) [inline src/components/layout/ContainerSections/Stocktray/index.tsx:36]: Adjust the name to PascalCase

File: src/components/layout/ContainerSections/Stocktray/index.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  const { isMobile, actionButtons, navigation } = props;
```
Comment (carin91) [inline src/components/layout/ContainerSections/Stocktray/index.tsx:37]: Include a storybook to the StockTray

File: src/components/layout/ContainerSections/index.tsx [hunk 3]
```diff
@@ -1,135 +1,28 @@
 import { useState } from "react";
```
Comment (carin91) [inline src/components/layout/ContainerSections/index.tsx:1]: Adjust the storybook which is broken

**Review by carin91 — APPROVED** [review]

File: src/components/layout/ContainerSections/Stocktray/index.tsx (hunk 1 above)
Comment (Andresbl123) [reply src/components/layout/ContainerSections/Stocktray/index.tsx:36]: Resuelto

File: src/components/layout/ContainerSections/Stocktray/index.tsx (hunk 2 above)
Comment (Andresbl123) [reply src/components/layout/ContainerSections/Stocktray/index.tsx:38]: Resuelto

File: src/components/layout/ContainerSections/index.tsx (hunk 3 above)
Comment (Andresbl123) [reply src/components/layout/ContainerSections/index.tsx]: resuelto

**Pull request merged**